sw/otbn/crypto/tests/generated/shake_testgen.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import random
+from typing import TextIO, Optional
+
+from Crypto.Hash import SHAKE128, SHAKE256
+
+from shared.testgen import write_test_data, write_test_exp
+
+DST_LEN = 32 # SHAKE128: 16, SHAKE256: 32
```

Comment:
Similar comment as for SHA3, should this be randomly selected instead of fixed so that SHAKE128 also gets tested?